dfds_ds_toolbox/analysis/plotting.py

```diff
@@ -144,45 +143,25 @@ def get_trend_stats(
 
 
 def plot_regression_predicted_vs_actual(
-    y_true: np.array, y_pred: np.array, title: str = "", extra_text: str = "", band_pct: float = 0.1
+    y_true: np.array, y_pred: np.array, alpha: float = 0.2
```

Comment:
In the other plotting functions we use Sequence for the typehints. Should we do the same in this one?

Reply:
Good point. Fixed